### PR TITLE
Patch: Add footer

### PR DESCRIPTION
**Pull request by ECWireless**
Builds off of #42 

- [x] Footer top layer
- [x] Footer bottom layer

### CODE REVIEW

File: src/components/Footer/components/index.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { media } from 'components/breakpoints';
+import { colors, GU } from 'components/theme';
+
+// MAIN CONTAINERS
```
Comment (Evalir) [inline src/components/Footer/components/index.ts]: Let's remove this comment :).

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import 'styled-components/macro';
+import { media } from 'components/breakpoints';
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as SelectorSvg } from 'assets/icons/selector.svg';
+import { ReactComponent as DiscrodSvg } from 'assets/icons/discord.svg';
```
Comment (Evalir) [inline src/components/Footer/index.tsx]: Hmm there seems to be a typo on `DiscrodSvg` 😛

File: src/components/Navigation/index.tsx [hunk 3]
```diff
@@ -46,22 +47,26 @@ const Navigation: React.FC<INavigation> = ({ readyToTransact, setSidebar }) => {
...
           <li>
-            <StyledLink exact activeClassName="active" to="/propose">
-              Propose App
-            </StyledLink>
-          </li>
-          <li>
-            <StyledLink exact activeClassName="active" to="/new-farm">
-              New Farm
+            <StyledLink exact activeClassName="active" to="/">
+              <div>
```
Comment (Evalir) [inline src/components/Navigation/index.tsx]: Hmm, I suppose the div is to force a `display: block` property for the SVGs—is there a way to do this (such as styling the link) without bloating the dom with unnecessary divs?

**Review by Evalir — COMMENTED** [review]
Looking good! A few comments:

File: src/components/Footer/components/index.ts (hunk 1 above)
Comment (ECWireless) [reply src/components/Footer/components/index.ts]: Fixed in bf544b3bed8d0c40e101bd75c3b0adaca5f84567

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (ECWireless) [reply src/components/Footer/index.tsx]: Fixed in 26fc03638439a3ab83c22444f4b60af2254954e8

File: src/components/Navigation/index.tsx (hunk 3 above)
Comment (ECWireless) [reply src/components/Navigation/index.tsx]: Indeed there is a better way! Fixed in ccf4b95e644f999cb1f4831ab5e7cc7e9b715a4f

**Review by Evalir — APPROVED** [review]
LGTM!

**Pull request merged**